### PR TITLE
Issue 10640: Some more inversed settings

### PR DESCRIPTION
**Pull request by annando**
See #10640

These are some more settings which now are inversed, so that they now enable and not disable the setting.

### CODE REVIEW

File: src/Module/Admin/Site.php [hunk 1]
```diff
@@ -163,9 +163,9 @@ function update_table(App $a, $table_name, $fields, $old_url, $new_url)
 		$allow_users_remote_self = !empty($_POST['allow_users_remote_self']);
 		$explicit_content       = !empty($_POST['explicit_content']);
 
-		$no_multi_reg           = !empty($_POST['no_multi_reg']);
-		$no_openid              = !empty($_POST['no_openid']);
-		$no_regfullname         = !empty($_POST['no_regfullname']);
+		$enable_multi_reg        = !empty($_POST['enable_multi_reg']);
+		$enable_openid           = !empty($_POST['enable_openid']);
+		$enable_regfullname     = !empty($_POST['enable_regfullname']);
```
Comment (MrPetovan) [inline src/Module/Admin/Site.php]:
```suggestion
		$enable_multi_reg       = !empty($_POST['enable_multi_reg']);
		$enable_openid          = !empty($_POST['enable_openid']);
		$enable_regfullname     = !empty($_POST['enable_regfullname']);
```

File: src/Module/Admin/Site.php [hunk 2]
```diff
@@ -555,9 +555,9 @@ public static function content(array $parameters = [])
...
-			'$no_openid'              => ['no_openid', DI::l10n()->t('Disable OpenID'), DI::config()->get('system', 'no_openid'), DI::l10n()->t('Disable OpenID support for registration and logins.')],
-			'$no_regfullname'         => ['no_regfullname', DI::l10n()->t('No Fullname check'), DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Allow users to register without a space between the first name and the last name in their full name.')],
+			'$enable_multi_reg'        => ['enable_multi_reg', DI::l10n()->t('Enable multiple registrations'), !DI::config()->get('system', 'block_extended_register'), DI::l10n()->t('Enable users to register additional accounts for use as pages.')],
+			'$enable_openid'           => ['enable_openid', DI::l10n()->t('Enable OpenID'), !DI::config()->get('system', 'no_openid'), DI::l10n()->t('Enable OpenID support for registration and logins.')],
+			'$enable_regfullname'     => ['enable_regfullname', DI::l10n()->t('Enable Fullname check'), !DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Enable check to only allow users to register with a space between the first name and the last name in their full name.')],
```
Comment (MrPetovan) [inline src/Module/Admin/Site.php]:
```suggestion
			'$enable_multi_reg'       => ['enable_multi_reg', DI::l10n()->t('Enable multiple registrations'), !DI::config()->get('system', 'block_extended_register'), DI::l10n()->t('Enable users to register additional accounts for use as pages.')],
			'$enable_openid'          => ['enable_openid', DI::l10n()->t('Enable OpenID'), !DI::config()->get('system', 'no_openid'), DI::l10n()->t('Enable OpenID support for registration and logins.')],
			'$enable_regfullname'     => ['enable_regfullname', DI::l10n()->t('Enable Fullname check'), !DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Enable check to only allow users to register with a space between the first name and the last name in their full name.')],
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Are these the last of the "Disable ..." settings or was it just for the Admin settings?

**Comment by annando** [comment]
I'm not totally sure with some settings, for example the "unlisted" setting in the user settings. I guess the best would be if someone else could also have a look.

**Comment by foss-** [comment]
Thanks for working on this. Display Settings > Content/Layout still has:

- Don't shot emoticons: Should be `Show emoticons` (Enabled by default)
- Disable smart threading: Should be `Enable smart threading` (Enabled by default)
- Hide the dislike feature: Should be `Show dislikes` (Enabled by default)

Slightly Off-topic, but what do you think about renaming `Content/Layout` to `Timeline Layout`? I don't think Word1/Word2 as a title for a section is good UX.

display settings   Content:Layout

**Pull request merged**